### PR TITLE
Alert dialog revised

### PR DESCRIPTION
**Pull request by DLabbate**
- Alert dialog now shows the status of the bluetooth sensor (Connected/Not Connected) and Location Services (Enabled/Not Enabled)

- Some UI cleanup (revised "Add Bike" dialog fragment to be consistent with other UI elements)

- Removed vibration from "RecordingService" background notification channel.

### CODE REVIEW

**Comment by DLabbate** [comment]
image
image

File: app/src/main/java/com/example/bikebuddy/RecordWorkoutDialog.java [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+            textViewLocationStatus.setText("Enabled");
+        }
+
+        else
+        {
+            textViewLocationStatus.setText("Not Enabled");
+        }
+    }
+
+    public static Boolean isLocationEnabled(Context context)
```
Comment (glepore98) [inline app/src/main/java/com/example/bikebuddy/RecordWorkoutDialog.java:93]: There's a method called isMapsEnabled in Login Activity that checks whether the GPs is enabled on the device. You could've used that

**Review by ArthurMoreDonuts — APPROVED** [review]
well done.

**Review by glepore98 — APPROVED** [review]

**Review by BDeslippe23 — APPROVED** [review]
Nice, For the alert when recording workout, would it be worthwhile to add a disclaimer that the data will be wrong if one of the services (bluetooth or location) are not available?

**Pull request merged**